dwave/plugins/torch/nn.py

```diff
@@ -0,0 +1,118 @@
+import inspect
+from functools import wraps
+from types import MappingProxyType
+
+import torch
+from torch import nn
+
+
```

Comment:
Don't forget to add an `__all__` here. See e.g., [here](https://github.com/dwavesystems/dwave-pytorch-plugin/blob/aa981186b1582c133019bdb7d0311b31f5228113/dwave/plugins/torch/models/boltzmann_machine.py#L46).